src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -284,7 +284,13 @@ public static string[] GetExcludedFiles(string[] excludes)
             const string RELATIVE_KEY = nameof(RELATIVE_KEY);
             string parentDir = Directory.GetCurrentDirectory();
 
-            if (excludes == null || !excludes.Any()) return Array.Empty<string>();
+            if (excludes is null || !excludes.Any()) 
```

Comment:
What happens via datacollector flow? If we have added a filter here [https://github.com/tonerdo/coverlet/blob/master/src/coverlet.collector/Utilities/CoverletConstants.cs#L21](https://github.com/tonerdo/coverlet/blob/master/src/coverlet.collector/Utilities/CoverletConstants.cs#L21),
will `excludes` be null?

Reply:
I see...I'm thinking for a simple solution.

Reply:
I think that I'll change strategies....I don't like add to core knowledge about who is the caller, could start a bad practices.
Go with duplication.